src/components/Question/Question.js

```diff
@@ -0,0 +1,87 @@
+import { html } from '../../shared';
+import styles from './Question.module.css';
+
+/**
+ * @typedef {Object} QuestionInstance
+ * @property {HTMLElement} question
+ * @property {() => boolean} isAnswerCorrect Function that returns whether the user has correctly answered a question
+ */
+
+/**
+ * @param {(import('../../data/questions').CharacterQuestion|import('../../data/questions').EpisodeOrLocationQuestion)} questionData
+ * @returns {QuestionInstance}
+ */
+function Question(questionData) {
+  const { category } = questionData;
+
+  const questionText = {
+    character: 'Co to za postać?',
+    episode: `Kto grał w odcinku ${questionData.name}?`,
+    // @ts-ignore
+    location: `Które postaci mieszkają w miejscu ${questionData.name}?`,
+  };
+
+  const answerLetter = {
+    0: 'A',
+    1: 'B',
+    2: 'C',
+    3: 'D',
+  };
```

Comment:
```suggestion
  const answerLetter = ['A', 'B', 'C', 'D'];
```
Myślę, że to na spokojnie może być tablica.

Reply:
O nie, to było takie oczywiste... Dzięki 😅